Assets/Scripts/PrefabScripts/Events/DoorScript.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Interfaces;
+
+public class DoorScript : MonoBehaviour, EventInterface
+{
+    private Vector3 startPosition;
+    private Vector3 endPosition;
+    public int speed;
+
+    private bool isTriggered = false;
+    void Start()
+    {
+        startPosition = transform.position;
+        endPosition = transform.position + new Vector3(0, 10, 0);
+        Debug.Log("Start Position: " + startPosition);
```

Comment:
Do we want all these logs in here?

Reply:
Delogging service has been and taken care of it.